### PR TITLE
add user login code

### PR DESCRIPTION
**Pull request by kkolamala1984**
added user login code

### CODE REVIEW

File: app/users/apps.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class UsersConfig(AppConfig):
+    name = 'users'
```
Comment (kolamala) [inline app/users/apps.py:5]: rename user to users

File: app/users/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```
Comment (kolamala) [inline app/users/tests.py:2]: add more tests

**Review by kolamala — CHANGES_REQUESTED** [review]
need changes

File: app/users/apps.py (hunk 1 above)
Comment (kkolamala1984) [reply app/users/apps.py:5]: renamed

File: app/users/tests.py (hunk 2 above)
Comment (kkolamala1984) [reply app/users/tests.py:2]: added more tests

**Pull request merged**